frb_rust/src/ffi.rs

```diff
@@ -0,0 +1,214 @@
+//! Stubs for allo_isolate on the web.
+
+#[cfg(not(target_family = "wasm"))]
+pub use allo_isolate::{ffi::DartCObject, *};
+#[cfg(not(target_family = "wasm"))]
+pub type MessagePort = i64;
+
+#[macro_export]
+macro_rules! console_log {
+    ($lit:literal) => {
+        #[allow(unused_unsafe)]
+        unsafe { $crate::ffi::log($lit) }
+    };
+    ($( $tt:tt )*) => {
+        #[allow(unused_unsafe)]
+        unsafe { $crate::ffi::log(&format!($( $tt )*)) }
+    };
+}
+
+#[macro_export]
+macro_rules! console_error {
+    ($lit:literal) => {
+        #[allow(unused_unsafe)]
+        unsafe { $crate::ffi::error($lit) }
+    };
+    ($( $tt:tt )*) => {
+        #[allow(unused_unsafe)]
+        unsafe { $crate::ffi::error(&format!($( $tt )*)) }
+    };
+}
```

Comment:
What is the difference between console_log and `info!`? Can we let `info!` automatically call console log (probably by injecting a log implementation)? Maybe there even already exists a crate for this

Reply:
Don't worry about these, they are just for internal debugging. (There is a crate for it, but JS bindings tend to weigh a lot.)

Reply:
I see :)